decide/src/main/java/group17/LicAnalyzer.java

```diff
@@ -271,7 +271,31 @@ public boolean lic10(InputHandler input) {
 
     }
 
+
+    
+    /** 
+     * This function calculates Launch Interceptor Condition (LIC) number 11
+     * 
+     * @param input
+     * @return boolean
+     */
     public boolean lic11(InputHandler input) {
+        int gpts = input.G_PTS;
+        int numpoints = input.NUMPOINTS;
+        double[] X = input.X_COORD;
+
+        if (gpts < 1) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 11. Reason: G_PTS must be greater than 1.");
+        } else if (gpts > numpoints - 2) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 11. Reason: G_PTS must be less than NUMPOINTS - 2.");
+        }
```

Comment:
You need to add change to check for NUMPOINTS. As of now, NUMPOINTS can be any value. Make sure to add that NUMPOINTS == 2 should return false and NUMPOINTS < 2 should throw an exception.